apps/website/src/pages/docs/stepper.mdx

```diff
@@ -9,19 +9,46 @@ group: core
 
 import PropsTable from '~/components/PropsTable.astro';
 import LiveExample from '~/components/LiveExample.astro';
-import Placeholder from '~/components/Placeholder.astro';
 import * as AllExamples from '~/examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='core-stepper' />
-
 <LiveExample src='Stepper.main.tsx'>
   <AllExamples.StepperMainExample client:load />
 </LiveExample>
 
 In defined, multi-step user interactions, it is useful to inform the user of the number of steps involved in the process, and the current step within that sequence. A UI 'Stepper' guides the user through the steps and provides some interaction to return to previous steps. A standard example of a Stepper would be an e-Commerce checkout process, but there are many others where the user must perform sequential steps.
 
+## Variants
+
+There are two different styles of stepper, depending on the amount of steps required to complete the process.
+
+### Short Stepper
+
```

Comment:
Should an example for a "short" stepper be shown here even if it's similar (if not identical) to the main example at the top of the page?

Reply:
I don't know what would be best to do here, and this will likely pop up in future #961 PRs.  I would like to hear @iTwin/itwinui thoughts on this.

If we don't get a clear direction from the comments, just do what you think is best and we can make sweeping changes once we make it to the end of documentation site content updates.

Reply:
I think the main example should be "the best version" of the component. The style of the stepper is defined by a prop and not automatically so I think we should have separate example of Short stepper just to say that this style is default.

Reply:
An example for short stepper has been added

Reply:
Could you do main and short example a bit more different? Maybe make short stepper to have only 3 steps or smth.

Reply:
The short example now only has 3 steps and the step labels are different from the main example